src/components/popover/Popover.js

```diff
@@ -39,7 +39,7 @@ const Popover = (props) => {
     returnFocusToSource,
   } = props;
 
-  const { ref, FocusRing } = useFocusTrap({ active, returnFocusToSource });
+  const { ref, FocusRing } = useFocusTrap({ active, returnFocusToSource, initialFocusRef: false });
```

Comment:
I'm still in doubt about this, it felt a bit better to not have an initial focus in popovers, since f.e. the split button has some interesting focus states:

<img width="404" alt="Screenshot 2021-07-12 at 09 22 44" src="https://user-images.githubusercontent.com/10011712/125246991-bfa29b80-e2f2-11eb-8ca8-8451fd667d76.png">

However, the focus will still be set the first focus bumper, which means that when you perform your first tab, the next focusable element in the popover will receive focus

Reply:
Hmm yeah, it depends a bit on the case I guess.
I recently built a Popover with some fields, and there it would make sense to receive an initial focus.
Maybe we should be able to overwrite it? Enable it by default, but disable it for some other components, e.g. the menu here. But then you could argue that it's not consistent...

Reply:
Planned a meeting to discuss this, in the meantime I don't think this is blocking for this story.